content/glass_elevator/_index.html

```diff
@@ -1,23 +0,0 @@
-+++
-type = "article"
-title = "Glass Elevator"
-date = "2006-12-18T20:43:48.000Z"
-tags = []
-+++
-
-<p>Glass Elevator was a project for discussing and documenting ideas for future Haiku versions, R2 and beyond.</p>
```

Comment:
this line should be preserved in one of the new files? otherwise there is no info left about what Glass Elevator is/was. And the name isn't quite self-describing

Reply:
Corrected in next commit.